app/(auth)/Login.tsx

```diff
@@ -0,0 +1,181 @@
+import React from "react";
+import { StyleSheet, Text, View, Image, TouchableOpacity } from "react-native";
+import Input from "@/components/Input";
+import Button from "@/components/Button";
+import ThirdPartyButton from "@/components/ThirdPartyLoginButton";
+import { router } from "expo-router";
+
+const Login = () => {
+  return (
+    <View style={styles.Container}>
+      <Image
+        style={styles.Logo}
+        source={require("../../assets/images/logo.png")}
+      />
+      <Text style={styles.PageTitle}>Login</Text>
+      <View style={styles.Row}>
+        <Text style={styles.Col1}>Don’t have an account?</Text>
+        <TouchableOpacity
+          onPress={() => {
+            router.push({
+              pathname: "/(auth)/Register",
+            });
+          }}
+        >
+          <Text style={styles.Col2}> Register here.</Text>
+        </TouchableOpacity>
+      </View>
+      <View style={styles.InputContainer}>
+        <Input inputMode="email" placeholder="Email address"></Input>
+        <Input inputMode="password" placeholder="Password"></Input>
+      </View>
+      <View style={styles.ForgotPasswordContainer}>
+        <TouchableOpacity>
+          <Text style={styles.Col2}>Forgot password?</Text>
+        </TouchableOpacity>
+      </View>
+      <Button
+        title="Login"
+        onPress={() => {
+          router.push({
+            pathname: "/(auth)/Register",
+          });
+        }}
+      />
+      <View style={styles.SpacerContainer}>
+        <View style={styles.Spacer}></View>
+        <Text style={styles.SpacerText}>Or continue with</Text>
```

Comment:
Text should be a separate component not imported from React native

Reply:
Done.